numpy/ma/core.py

```diff
@@ -5223,6 +5223,12 @@ def mean(self, axis=None, dtype=None, out=None, keepdims=np._NoValue):
         numpy.mean : Equivalent function
         numpy.ma.average : Weighted average.
 
+        Notes
+        -----
+        The default size of the intermediate and return values used for
+        integer inputs is platform-dependent. This is in contrast from
```

Comment:
If I understand correctly, this behaviour occurs only when the integer input `dtype` is of less precision than the default platform integer. From the docs of `numpy.sum()`:

> The dtype of a is used by default unless a has an integer dtype of less precision than the default platform integer. In that case, if a is signed then the platform integer is used while if a is unsigned then an unsigned integer of the same precision as the platform integer is used.

This might be worth mentioning. Otherwise, LGTM. Thanks!